source/index.md

```diff
@@ -33,6 +33,12 @@ members:
   name: The **Global Project ValuES - Methods for Integrating Ecosystem Services into Policy, Planning, and Practice**, implemented by GIZ on behalf of BMU, as part of the German Government’s International Climate Initiative (IKI) 
   website: http://aboutvalues.net/
   background: values
+ icsd-kaz:
+  name: The **Interstate Commission on Sustainable Development (Scientific and Information Center Kazakhstan)** 
+  website: http://www.mkurca.org/
+  microsite: https://kz.icsd.klink.asia/projects/rpoosur/ru
```

Comment:
@shenriod the attribute `microsite` is not valid. Only `kbox` and `website` are used. The `kbox` attribute is usually used for navigating to the login page as a shortcut for users, while the `website` attribute is used to direct the visitor to the official description of the specific member. If you want to direct traffic to the microsite you could insert that URL into the `kbox` attribute.

Reply:
Done